solr/solrj/src/java/org/apache/solr/client/solrj/impl/Http2SolrClient.java

```diff
@@ -266,6 +267,7 @@ private HttpClient createHttpClient(Builder builder) {
         asyncTracker.getMaxRequestsQueuedPerDestination());
     httpClient.setUserAgentField(new HttpField(HttpHeader.USER_AGENT, AGENT));
     httpClient.setIdleTimeout(idleTimeoutMillis);
+    httpClient.setCookieStore(new HttpCookieStore.Empty());
```

Comment:
So another big consideration here besides authentication - is load balancers also use cookies to ensure there client goes back to the same backend endpoint.

Reply:
Few references:

* https://aws.amazon.com/about-aws/whats-new/2021/02/application-load-balancer-supports-application-cookie-stickiness/
* https://www.f5.com/services/resources/white-papers/cookies-sessions-and-persistence

Reply:
Ah you are right. Maybe it makes sense to have it as an option in the builder, WDYT?

Reply:
Makes sense to me